bin/MadGraph5_aMCatNLO/cards/production/13TeV/HeavyNeutrino_trilepton_V-0p01_tau_NLO/HeavyNeutrino_trilepton_mNgtmZ_V-0p01_tau_NLO/HeavyNeutrino_trilepton_V-0p01_tau_NLO_customizecards.dat

```diff
@@ -1,4 +1,4 @@
-set param_card mass 9900012 1000
+set param_card mass 9900012 HNLMASS
```

Comment:
as above. what about the width ?

Reply:
Same, we recompute the width in customizecards